omnibus/config/projects/chef.rb

```diff
@@ -51,15 +51,27 @@
 # Chef Release version pinning
 override :chef, version: "local_source"
 override :ohai, version: "master"
+override :"rb-readline", version: "v0.5.3"
```

Comment:
I left the default version pointing to master like a lot of other software definitions do in omnibus-software.  It felt like the natural "default" for anyone wanting rb-readline and makes the version used here explicit.  I can swap it out if that's more sensible.